remappers/kubernetesmetrics/clustermetrics.go

```diff
@@ -0,0 +1,52 @@
+package kubernetesmetrics
+
+import (
+	remappers "github.com/elastic/opentelemetry-lib/remappers/internal"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+func addClusterMetrics(
+	src, out pmetric.MetricSlice,
+	_ pcommon.Resource,
+) error {
+	var timestamp pcommon.Timestamp
+	var node_allocatable_memory, node_allocatable_cpu int64
+
+	// iterate all metrics in the current scope and generate the additional Elastic kubernetes integration metrics
+	dataset := "kubernetes.node"
+	for i := 0; i < src.Len(); i++ {
+		metric := src.At(i)
+		if metric.Name() == "k8s.node.allocatable_cpu" {
+			dp := metric.Gauge().DataPoints().At(0)
+			if timestamp == 0 {
+				timestamp = dp.Timestamp()
+			}
+			node_allocatable_cpu = dp.IntValue()
+		} else if metric.Name() == "k8s.node.allocatable_memory" {
+			dp := metric.Gauge().DataPoints().At(0)
+			if timestamp == 0 {
+				timestamp = dp.Timestamp()
+			}
+			node_allocatable_memory = dp.IntValue()
+		}
+	}
+
+	remappers.AddMetrics(out, dataset, remappers.EmptyMutator,
+		remappers.Metric{
+			DataType:  pmetric.MetricTypeGauge,
+			Name:      "kubernetes.node.cpu.allocatable.cores",
```

Comment:
Could we define these as constants as well and maybe define the "remapping" definition in a golang map? If the map makes the result too ugly then we can just go with the constants.

Reply:
Ok I moved them all in main file https://github.com/elastic/opentelemetry-lib/pull/20/files#diff-363de0029a58d415b072beaa08a95048f1ce0310f29949d1a5e1ecd4176d36afR29

Following the structure of hostmetrics.

I think now is ok

Reply:
It seems that only the datasets where moved not the metric names 🤔